src/test/fuzz/util.cpp

```diff
@@ -204,18 +204,13 @@ void FillNode(FuzzedDataProvider& fuzzed_data_provider, CNode& node, bool init_v
     const ServiceFlags remote_services = ConsumeWeakEnum(fuzzed_data_provider, ALL_SERVICE_FLAGS);
     const NetPermissionFlags permission_flags = ConsumeWeakEnum(fuzzed_data_provider, ALL_NET_PERMISSION_FLAGS);
     const int32_t version = fuzzed_data_provider.ConsumeIntegralInRange<int32_t>(MIN_PEER_PROTO_VERSION, std::numeric_limits<int32_t>::max());
-    const bool filter_txs = fuzzed_data_provider.ConsumeBool();
 
     node.nServices = remote_services;
     node.m_permissionFlags = permission_flags;
     if (init_version) {
         node.nVersion = version;
         node.SetCommonVersion(std::min(version, PROTOCOL_VERSION));
     }
-    if (node.m_tx_relay != nullptr) {
-        LOCK(node.m_tx_relay->cs_filter);
-        node.m_tx_relay->fRelayTxes = filter_txs;
```

Comment:
eeb0e84fc5c259dfd00f7673ad4a51582b100282: Would be nice to not break the fuzz test. Maybe a test-only mock method on PeerMan can do this?

Reply:
Does this break the fuzz test? `fRelayTxes` (`m_relay_txs` after this PR) will get set after the node receives and processes the `version` message. Presumably the fuzzer will be able to set the `fRelay` field on that message to true or false.

Reply:
Thanks, fixed in https://github.com/bitcoin/bitcoin/pull/23575